examples/orpheus/finetune.yml

```
@@ -0,0 +1,52 @@
+base_model: canopylabs/orpheus-tts-0.1-pretrained
+
+hub_model_id: <your-hub-model-id>
+
+plugins:
+  - axolotl.integrations.liger.LigerPlugin
+liger_rope: true
+liger_rms_norm: true
+liger_glu_activation: true
+liger_fused_linear_cross_entropy: true
+
+datasets:
+  - path: <your-hf-dataset-id>
+    type: 
```

Should we leave a comment here, so user's don't accidentally think they need to put sth for `type: `?

```suggestion
    type:  # leave empty to load pre-tokenized
```